### PR TITLE
signals: addition of signal before info request

### PR DESCRIPTION
**Pull request by blixhavn**
* NEW Adds signal before IIIFImageInfo get request.

Signed-off-by: Øystein Blixhavn <oystein@blixhavn.no>

### CODE REVIEW

**Comment by drjova** [comment]
@blixhavn could you please add also ``iiif_before_after_request``?

**Comment by drjova** [comment]
and please update the copyright with 2017 :)

**Comment by blixhavn** [comment]
`iiif_before_after_request`? Do you mean `iiif_after_info_request`?

**Comment by drjova** [comment]
yep, sorry for the typo :D

File: tests/test_restful_api_signals.py [hunk 1]
```diff
@@ -54,3 +61,25 @@ def test_api_signals(self):
...
+
+        with signal_listener(iiif_after_info_request) as listener:
+            response = self.get(
+                'iiifimageinfo',
+                urlargs=infodata
+            )
+            results = listener.assert_signal()
+            self.assertEqual(
+                results[1],
+                json.loads(response.data.decode('utf-8'))
```
Comment (drjova) [inline tests/test_restful_api_signals.py]: @blixhavn just that and is ready, ``response.get_data(as_text=True)``

Comment (blixhavn) [inline tests/test_restful_api_signals.py]: Ah, nice

**Comment by blixhavn** [comment]
@drjova Done :)

**Pull request merged**